src/app/backing/components/AllocationBar/AllocationBarTooltip.tsx

```diff
@@ -0,0 +1,38 @@
+import { motion, AnimatePresence } from 'motion/react'
+import { useState } from 'react'
+import { b } from 'vitest/dist/chunks/suite.d.FvehnV49.js'
+
+interface AllocationBarTooltipProps {
+  builderAddress: string
+  currentBacking: number
```

Comment:
we should include a pending backing along with the current pending and they should be both bigint